scripts/barclamp_install.rb

```diff
@@ -188,7 +189,13 @@ def rm_tmpdir(tmpdir)
     end
     debug "installing barclamp"
     begin
-      bc_install from_rpm, bc[:name], bc[:src], bc[:yaml]
+      if bc[:yaml]["crowbar"]["layout"].to_i == 1
+        debug "Installing app components"
+        bc_install_layout_1_app from_rpm, bc[:name], bc[:src], bc[:yaml]
+        bc_install_layout_1_chef_migrate bc[:name], bc[:migrate], log
+      else
+        raise "ERROR: could not install barclamp #{bc[:name]} because #{bc[:yaml][:barclamp][:crowbar_layout]} is unknown layout."
```

Comment:
Should we really `raise`? Since some barclamps will have been installed, maybe we should only output an error and go on?